cart setup

Kuchvara opened the pull request:
Tried to do it completely by myself but anyway took something from video tutorials

Kuchvara committed:
cart setup

vhamolia commented:
Good idea, but errors in console https://prnt.sc/1v64tqk on load Shop and on click to cart btn.

Kuchvara committed:
mega fix

Kuchvara committed:
some fixes

cart item increase and decrease fixes (through local storage refreshing) + editing cart item remove btn

vhamolia commented:
Good job!
